app/src/main/java/io/mochahub/powermeter/data/workoutsession/WorkoutSessionDao.kt

```diff
@@ -15,7 +15,7 @@ interface WorkoutSessionDao {
     fun getAll(): Flow<List<WorkoutSessionWithRelation>>
 
     @Query("SELECT * FROM workout_sessions WHERE id = :workoutSessionID")
-    suspend fun find(workoutSessionID: String): WorkoutSessionEntity
```

Comment:
Is this necessary?

Reply:
It will return null if the workout session doesn’t exist.